server/backend/api/app/routers/router_smoke_test.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from typing import List
+from api.app.crud import crud_application
+from fastapi import APIRouter, Depends, Response
+from sqlalchemy.orm import Session
+from .. import database
+from api.app.models import model as models
+
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("", status_code=200)
+def get_applications(
```

Comment:
Seems weird to have the method for smoke testing called get_applications

Reply:
Good point. I'll rename it.

Reply:
Renamed